astropy/modeling/functional_models.py

```diff
@@ -33,6 +35,17 @@ class BaseGaussian1D(Fittable1DModel):
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
 
+    def __init__(self, amplitude, mean, stddev, **kwargs):
+        # Ensure stddev makes sense if its bounds are not explicitly set.
+        if 'bounds' not in kwargs:
+            kwargs['bounds'] = {'stddev': (FLOAT_EPSILON, None)}
```

Comment:
You should add a comment here and for the 2D Gaussian to explain why not simply ``(0, None)``, just for anyone looking the code in future.